Update npm-shrinkwrap.json

rodrigoprimo opened the pull request:
Update npm-shrinkwrap.json to update chromedriver that was updated by PR #31 

cc @bsessions85

rodrigoprimo committed:
Update npm-shrinkwrap.json

Update npm-shrinkwrap.json to update chromedriver that was updated by PR #31

rodrigoprimo commented:
The Travis build for this PR failed with the following error:

`SessionNotCreatedError: session not created exception: Chrome version must be >= 63.0.3239.0`

I'm trying to understand why the build for this PR failed and the build for PR #31 passed. I'm assuming that we need to update npm-shrinkwrap.json to make sure that packages that depend on this one install the correct dependencies version (in other words, the correct chromedriver version in this case specifically).

bsessions85 commented:
It doesn't make sense to me that it would fail either. I would probably update .travis.yml to use Chrome like:
 ```
sudo: required
dist: trusty
language: node_js
addons:
  chrome: stable
node_js:
  - "6"
before_script:
  - "export DISPLAY=:99.0"
  - "sh -e /etc/init.d/xvfb start"
  - sleep 3
script:
  - npm run lint
  - npm run clean
  - npm run build
  - npm run test
branches:
  only:
    - master

rodrigoprimo committed:
Add Chrome to .travis.yml

Doing this as an attempt to fix the following error in the Travis build (see https://github.com/woocommerce/wp-e2e-webdriver/pull/32#issuecomment-373157831):

SessionNotCreatedError: session not created exception: Chrome version must be >= 63.0.3239.0

rodrigoprimo commented:
Your suggestion fixed the issue, @bsessions85. Thanks!
